### PR TITLE
doc:update acrn_configuration_tool

### PR DESCRIPTION
**Pull request by lkanx-0811**
Signed-off-by: lkanx <lix.kan@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

**Comment by lkanx-0811** [comment]
OK to verify

File: doc/tutorials/acrn_configuration_tool.rst [hunk 1]
```diff
@@ -90,7 +92,8 @@ As an input for the ``acrn-config`` tool, end users do not need to care about th
 
 Scenario XML format
 ===================
-
+The board XML located at:
```
Comment (szhen11) [inline doc/tutorials/acrn_configuration_tool.rst]: should be "The Scenario XML located at"

File: doc/tutorials/acrn_configuration_tool.rst [hunk 2]
```diff
@@ -155,7 +158,8 @@ Additional scenario XML elements:
 
 Launch XML format
 =================
-
+The board XML located at:
```
Comment (szhen11) [inline doc/tutorials/acrn_configuration_tool.rst]: should be "The launch XML is located at"

File: doc/tutorials/acrn_configuration_tool.rst [hunk 3]
```diff
@@ -55,7 +55,7 @@ to launch post-launched User VMs.
 
 Scenario based VM configurations are organized as ``*.c/*.h`` files located at::
 
-   acrn-hypervisor/hypervisor/scenarios/$(SCENARIO)/
+   acrn-hypervisor/hypervisor/scenarios/$(BOARD)/
```
Comment (szhen11) [inline doc/tutorials/acrn_configuration_tool.rst]: should be "acrn-hypervisor/hypervisor/scenarios/$(SCENARIO)/"

**Pull request merged**